common-lib/src/main/scala/com/gu/mediaservice/lib/config/MetadataConfig.scala

```diff
@@ -114,7 +114,19 @@ object MetadataConfig {
     "Suki Dhanda"         -> "The Observer"
   )
 
-  val contractIllustrators = List("Winshluss", "Robert Crumb", "Steve Bell")
+  val contractIllustrators = List(
+    "Robert G Fresson",
+    "Bill Bragg",
+    "Sebastien Thibault",
+    "Eva Bee",
+    "Nate Kitch",
+    "Joe Magee",
+    "Jasper Rietman",
+    "Matt Kenyon",
+    "Andreij Krauze",
+    "Ellie Foreman-Peck",
+    "Ben Jennings"
```

Comment:
Gee... It's Andrzej Krauze, not Andreij! Of e.g. this less famous one http://grafik.rp.pl/grafika2/817291,9.jpg. I haven't checked spelling on the others, though. What about Observer's, like e.g. Chris Riddell?